lib/js/src/components/text.js

```diff
@@ -55,9 +56,10 @@ function CreateComponent(Impl) {
 
 var Impl = /* module */[/* Text */ReactNative.Text];
 
-function make(accessible, allowFontScaling, ellipsizeMode, numberOfLines, onLayout, onLongPress, onPress, pressRetentionOffset, selectable, style, testID, selectionColor, textBreakStrategy, adjustsFontSizeToFit, minimumFontScale, suppressHighlighting, value, children) {
+function make(accessible, accessibilityLabel, allowFontScaling, ellipsizeMode, numberOfLines, onLayout, onLongPress, onPress, pressRetentionOffset, selectable, style, testID, selectionColor, textBreakStrategy, adjustsFontSizeToFit, minimumFontScale, suppressHighlighting, value, children) {
```

Comment:
You forgot to build after removing `accessibilityLabel` from `Text`. The rest is awesome. Good job

Reply:
Ah nice catch, spaced out on that one. Fixed in: d0ccfc1